x-pack/test/observability_functional/apps/observability/pages/alert_details_page.ts

```diff
@@ -29,7 +29,7 @@ export default ({ getService }: FtrProviderContext) => {
     });
 
     it('should show 404 page when the feature flag is disabled', async () => {
-      await observability.alerts.common.navigateToAlertDetails(uuid.v4(), uuid.v4());
+      await observability.alerts.common.navigateToAlertDetails(uuid.v4());
```

Comment:
nit: In general, it is a best practice to have the waiting for the loading of the page inside the navigation function (`navigateToAlertDetails` in this case), and then here, we don't need to wait anymore and we can assert what we expect :)

Reply:
I checked the code. There is actually waiting for loading of the page inside `navigateToAlertDetails` function. I am not sure why there is waiting in the test case. I will have a look in more depth when I add functional tests for this component.